WORKSPACE

```diff
@@ -146,3 +146,27 @@ load(
 nvshmem_redist_init_repository(
     nvshmem_redistributions = NVSHMEM_REDISTRIBUTIONS,
 )
+http_archive(
```

Comment:
is it possible to put these rules inside a macro, for example, `sycl_init_repositories` ?